### PR TITLE
:sparkles: remove a dice

### LINKED ISSUE #18: Remove only one dice

### PR DESCRIPTION
**Pull request by fabienjuif**
fixes #18

### CODE REVIEW

File: src/contexts/dices.jsx [hunk 1]
```diff
@@ -38,13 +38,24 @@ const DicesProvider = ({ children }) => {
     return dice
   }
 
+  const remove = id => (e) => {
+    e.stopPropagation()
```
Comment (delphinemillet) [inline src/contexts/dices.jsx]: why is this needed ?

Comment (fabienjuif) [reply src/contexts/dices.jsx]: I forget to remove it.
In the first version I wrote I added the remove button in the dice button.

But it was not clever.
I changed to have a `div` representing the whole dice, with "roll" button + "remove" button as siblings, but I forgot to remove this.

File: src/contexts/dices.jsx [hunk 2]
```diff
@@ -38,13 +38,24 @@ const DicesProvider = ({ children }) => {
     return dice
   }
 
+  const remove = id => (e) => {
```
Comment (fabienjuif) [inline src/contexts/dices.jsx]:
```suggestion
  const remove = id => () => {
```

File: src/contexts/dices.jsx (hunk 1 above)
Comment (fabienjuif) [inline src/contexts/dices.jsx]:
```suggestion
```

File: src/contexts/dices.jsx [hunk 3]
```diff
@@ -38,13 +38,24 @@ const DicesProvider = ({ children }) => {
...
+    e.stopPropagation()
+
```
Comment (fabienjuif) [inline src/contexts/dices.jsx]:
```suggestion
```

**Review by delphinemillet — APPROVED** [review]
